### ISSUE TITLE
Provide Importable PrismaService

### ISSUE THREAD

**Issue by iddan** [marcjulian/nestjs-prisma#2]
For our use case, we don't need to customize the `PrismaService`. Can this package provide simple importable `PrismaService` and `PrismaModule`?

**Comment by marcjulian** [comment]
@iddan I added a `PrismaModule` in the latest release [0.5.0](https://github.com/marcjulian/nestjs-prisma/releases/tag/v0.5.0). Does this cover your use case?

**Comment by iddan** [comment]
Thank you for the quick triage! This is better, but I would like to import PrismaService and PrismaModule directly from the package instead of code generating them into my source code. Can that be possible?

**Comment by marcjulian** [comment]
It is possible, but it was designed at the moment just as a schematics module. But makes sense to include the `PrismaService` and `PrismaModule` as they don't require any adjustment usually.

**Comment by iddan** [comment]
Right, I think those are two use cases the library can support

**Comment by marcjulian** [comment]
@iddan Can you please test the new dev version `0.6.0-dev.2` 

You can run 

```bash
nest add nestjs-prisma@dev
```

This package now also exports `PrismaModule` and `PrismaService`.

```ts
import { PrismaModule } from 'nestjs-prisma';
import { PrismaService } from 'nestjs-prisma';
```

**Comment by iddan** [comment]
Great stuff! Works perfectly.

**Comment by iddan** [comment]
Can you publish it as a stable version?

**Comment by iddan** [comment]
Thank you!

**Comment by marcjulian** [comment]
No worries :) will be in the next version

**Comment by marcjulian** [comment]
@iddan I released [0.6.0](https://github.com/marcjulian/nestjs-prisma/releases/tag/v0.6.0) and it includes your great suggestion! Keep your feedback coming 👍
